files/SZWESAMP/ZWEIAPF

```diff
@@ -0,0 +1,30 @@
+//ZWEIAPF JOB
+//*                                                                    
+//* This program and the accompanying materials are made available     
+//* under the terms of the Eclipse Public License v2.0 which          
+//* accompanies this distribution, and is available at                 
+//* https://www.eclipse.org/legal/epl-v20.html                         
+//*                                                                    
+//* SPDX-License-Identifier: EPL-2.0                                   
+//*                                                                    
+//* Copyright Contributors to the Zowe Project. 2020, 2020             
+//*                                                                    
+//*********************************************************************
+//*
+//*  This JCL is used to set APF for the two datasets of Zowe
+//*  Which need it. You can issue this, or use another
+//*  Way to accomplish the task.
+//*
+//*********************************************************************
+//*
+//*  This dataset holds the APF portion of Zowe
+//         SET LOADLIB='{zowe.setup.dataset.authLoadlib}'
+//*
+//*  This dataset holds product plugins for ZIS,
+//*  ZIS is located in the LOADLIB.
+//         SET PLUGINLIB='{zowe.setup.dataset.authPluginLib}'
```

Comment:
### Exec missing
It seems, there must be some exec like `//EXEC EXEC PGM=IEFBR14`
### Name too long: PLUGINLIB
Impossible to find in IBM doc, but based on quick testing, name is probably up to 8 chars (as expected on mainframe):
```
STMT NO. MESSAGE                                                          
       4 IEFC642I EXCESSIVE PARAMETER LENGTH IN THE SYMBOLIC PARAMETER    
```
### Substitution not working
And this was in syslog:
```
09:26:19.17 INSTREAM 00000290  SETPROG APF,ADD,DSN=&LOADLIB   
09:26:19.18 INSTREAM 00000290  SETPROG APF,ADD,DSN=&PLUGLIB   
```
I am not sure if the substitution in `COMMAND` is even possible (no IBM doc examples or hints).

Reply:
* exec missing
Are you sure about this? EXEC is a 'statement', and COMMAND is a different 'statement'
https://www.ibm.com/docs/en/zos/2.1.0?topic=statements-jcl

* substitution not working
if i can find how to line-wrap this, i can get rid of the SET
but i'm not sure how yet. basically, we need to experiment and i havent used a test system with such permissions to do the experiment yet.

worst case... go back to using shell, but in the form of BPXBATCH jcl?